sidecar_acceptor.go

```diff
@@ -617,8 +595,7 @@ func (a *SidecarAcceptor) matchPrepare(pendingBatch *order.Batch,
 	// that's being used to pay for the sidecar channel.
 	err = a.cfg.FundingManager.PrepChannelFunding(batch, a.getSidecarAsOrder)
 	if err != nil {
-		return nil, fmt.Errorf("error preparing channel funding: %v",
-			err)
+		return fmt.Errorf("error preparing channel funding: %v", err)
```

Comment:
We need to wrap this error, otherwise the `errors.As(failure, &partialReject):` below won't trigger.